puku.go

```diff
@@ -85,7 +85,7 @@ var funcs = map[string]func(conf *config.Config, plzConf *please.Config, orignal
 }
 
 func main() {
-	flags.ParseFlagsOrDie("puku", &opts, nil)
+	cmd := flags.ParseFlagsOrDie("puku", &opts, nil)
```

Comment:
we were double parsing which meant the paths were doubled up. This broke the migrator.